translator/GCP_cloud_function/cloud_function/main.py

```diff
@@ -1,35 +1,53 @@
-# {'bucket': 'icone_files', 'contentType': 'image/jpeg', 'crc32c': 'ZO4N6A==', 'etag': 'CNu+up7tnu4CEAE=', 
-# 'generation': '1610745138552667', 'id': 'icone_files/61862931_1039122999618882_7010636350885986304_o_1039122996285549.JPEG/1610745138552667', 
-# 'kind': 'storage#object', 'md5Hash': 'yOlwCtFi0L/3GPe23rYGbw==', 
-# 'mediaLink': 'https://www.googleapis.com/download/storage/v1/b/icone_files/o/61862931_1039122999618882_7010636350885986304_o_1039122996285549.JPEG?generation=1610745138552667&alt=media', 
-# 'metageneration': '1', 'name': '61862931_1039122999618882_7010636350885986304_o_1039122996285549.JPEG',
-#  'selfLink': 'https://www.googleapis.com/storage/v1/b/icone_files/o/61862931_1039122999618882_7010636350885986304_o_1039122996285549.JPEG',
-#   'size': '447421', 'storageClass': 'STANDARD', 'timeCreated': '2021-01-15T21:12:18.552Z', 'timeStorageClassUpdated': '2021-01-15T21:12:18.552Z', 
-#   'updated': '2021-01-15T21:12:18.552Z'}
-
 import icone_translator
 from gcloud import storage
 import json
 import xmltodict
+import shutil
+import urllib.request as request
+from contextlib import closing
+from google.cloud import pubsub_v1
+
+
+server = 'iconetraffic.com'
+port = '42663'
+user = 'cdot'
+password = 'icone_cdot'
```

Comment:
Any potentially sensitive information, such as username and password, should not be hard coded in here. Instead these should be coming from a secure location, try [Secrets](https://dev.to/googlecloud/using-secrets-in-google-cloud-functions-5aem).

Reply:
We did not have permissions to access secrets ,but we will pursue it in next sprint.